### ISSUE TITLE
tests are not actually run in CI, also some of them don't build anymore

### ISSUE THREAD

**Issue by yomimono** [mirage/ocaml-dns#150]
See [this build log](https://travis-ci.org/yomimono/ocaml-dns/jobs/332943670), where one can find many removals and installs but not a single test run.  This isn't too surprising once one notes that there are no `build-test` stanzas in any of the `opam` files, nor are there any declared test dependencies (although the tests do have dependencies declared in their respective `jbuild`s in `lib_test`).  It looks like at least some of the tests are no longer buildable, also:

```
$ jbuilder build @runtest
...
File "lib_test/async/test_async_dns_resolver_unix.ml", line 30, characters 45-49:
Error: The function applied to this argument has type
         ?extract_exn:bool -> ?readme:(unit -> string) -> Core__Command.t
This argument cannot be applied without label
```

If this is intentional and there is some alternative testing paradigm at work (or simply no desire to use Travis for this purpose), please feel free to close this issue.

**Comment by avsm** [comment]
Good catch!

**Comment by avsm** [comment]
fixed in 2a5def03b6b9c9fe8c9fcfee346b06aa180eac24
